Core/Dialog/PSPDialog.cpp

```diff
@@ -31,7 +31,7 @@
 #define FADE_TIME 1.0
 const float FONT_SCALE = 0.55f;
 
-PSPDialog::PSPDialog() {
+PSPDialog::PSPDialog(int type) : dialogType_(type) {
```

Comment:
`type` here (and `dialogType_`) should probably just be a `UtilityDialogType` instead of `int`? same in all the dialog classes?

Reply:
Well, I was just too lazy to move it into the header.

-[Unknown]

Reply:
I think it's (just barely) worth doing - if you don't feel like it, I will after merge ;)